src/VisualStudio/IntegrationTest/IntegrationTests/CSharp/CSharpInteractiveDirectives.cs

```diff
@@ -17,8 +17,8 @@ public CSharpInteractiveDirectives(VisualStudioInstanceFactory instanceFactory)
         [Fact]
         public void VerifyHostCommandsCompletionList()
         {
-            VisualStudio.Workspace.SetUseSuggestionMode(true);
             VisualStudio.InteractiveWindow.InsertCode("#");
+            VisualStudio.Workspace.SetUseSuggestionMode(true);
```

Comment:
hmmm, this indicates some important difference. Seems like the editor cannot find proper buffer to execute "ToggleSuggestionMode" command before "#" is typed, while Roslyn could.

Reply:
fixed the issue in the editor bits so now reverting this change back